boards/nordic/thingy52/thingy52_nrf52832.dts

```diff
@@ -37,17 +37,20 @@
 
 	leds {
 		compatible = "gpio-leds";
+
 		/* Lightwell RGB */
 		led0: led_0 {
 			gpios = <&sx1509b 7 GPIO_ACTIVE_LOW>;
 			label = "Red LED";
 			//vin-supply = <&vdd_pwr>;
```

Comment:
Can these go?

Reply:
(//vin-supply . . . piece).

Reply:
Seems added by a 5 year old commit to signify that the power from the LEDs is coming from vdd_pwr as part of the sx1509b I/O expander, assuming just as a comment to indicate, but no explicit reason to keep it